app/client/src/features/events/Moderation/LiveFeedbackPromptResponses/LiveFeedbackPromptResponseList.tsx

```diff
@@ -0,0 +1,196 @@
+                            <Card>
+                                <CardContent>
+                                    <Grid container direction='row' alignItems='center' justifyContent='space-around'>
+                                        <Grid item>
+                                            <Typography
+                                                variant='inherit'
+                                                sx={{ wordBreak: 'break-word' }}
+                                                color={promptData.isVote ? getVoteColor(vote) : 'black'}
+                                            >
+                                                {response}
+                                            </Typography>
+                                        </Grid>
+                                    </Grid>
+                                </CardContent>
+                            </Card>
+                        </Grid>
+                    </ListItem>
+                ))}
+            </List>
+        </React.Fragment>
+    );
+}
+
+interface LiveFeedbackPromptResponseListProps {
+    queryRef: PreloadedQuery<LiveFeedbackPromptResponseListQuery>;
+    promptData: PromptData;
+}
+
+/**
+ * This component is responsible for loading the query and passing the fragment ref to the PromptList component
+ */
+export function LiveFeedbackPromptResponseList({ queryRef, promptData }: LiveFeedbackPromptResponseListProps) {
+    const { promptResponses } = usePreloadedQuery(LIVE_FEEDBACK_PROMPT_RESPONSE_LIST_QUERY, queryRef);
+    if (!promptResponses) return <Loader />;
+    return <PromptResponseList promptResponses={promptResponses} promptData={promptData} />;
+}
+
+interface PreloadedLiveFeedbackPromptResponseListProps {
+    promptData: PromptData;
+}
+
+export function PreloadedLiveFeedbackPromptResponseList({ promptData }: PreloadedLiveFeedbackPromptResponseListProps) {
+    const [queryRef, loadQuery] = useQueryLoader<LiveFeedbackPromptResponseListQuery>(
+        LIVE_FEEDBACK_PROMPT_RESPONSE_LIST_QUERY
+    );
+    const [isRefreshing, setIsRefreshing] = React.useState(false);
+    const { env } = useEnvironment();
+    const REFRESH_INTERVAL = 10000; // 10 seconds
+    const { promptId } = promptData;
+
+    const refresh = React.useCallback(() => {
+        if (isRefreshing) return;
+        setIsRefreshing(true);
+        fetchQuery(env, LIVE_FEEDBACK_PROMPT_RESPONSE_LIST_QUERY, { promptId }).subscribe({
+            complete: () => {
+                setIsRefreshing(false);
+                loadQuery({ promptId }, { fetchPolicy: 'store-or-network' });
+            },
+            error: () => {
+                setIsRefreshing(false);
+            },
+        });
+    }, [env, isRefreshing, loadQuery, promptId]);
+
+    React.useEffect(() => {
+        if (!queryRef) loadQuery({ promptId });
+        const interval = setInterval(refresh, REFRESH_INTERVAL);
+        return () => clearInterval(interval);
+    }, []);
```

Comment:
Should export the refresh and call effect in parent component to avoid strange behavior